docs/recipes/parcel-react.mdx

```diff
@@ -0,0 +1,173 @@
+import { ChromeExtension } from '../../components/extension';
+import { Image } from '../../components/ui/image';
+import { SourceCode, Tabs, Tab } from '@theme';
+
+# Parcel with Zephyr
+
+<SourceCode href="https://github.com/ZephyrCloudIO/zephyr-examples/tree/main/examples/parcel-app" />
+
+In this guide, we'll walk through creating a Parcel application and configuring it to deploy with Zephyr.
+
+:::info Prerequisites
+- We expect you to have finished our [Get Started](/general/get-started) guide.
+- Have our [browser extension](https://chromewebstore.google.com/detail/zephyr-mission-control/liflhldchhinbaeplljlplhnbkdidedn) installed in your browser.
+- A registered account on [Zephyr Cloud](https://app.zephyr-cloud.io).
+
+If you don't have `nvm` to manage your node version yet, head to [the official nvm guide](https://github.com/nvm-sh/nvm) to install it.
+```bash
+nvm use 20 
+```
+:::
+
+## Creating a New Project
+
+The easiest way to get started is using the `create-zephyr-apps` CLI:
+
+<Tabs values={[{label:"npm"}, {label: 'yarn'}, {label: 'pnpm'}]}>
+<Tab>
+```bash filename="Terminal"
+npx create-zephyr-apps@latest
+```
+</Tab>
+<Tab>
+```bash filename="Terminal"
+yarn create zephyr-apps
+```
+</Tab>
+<Tab>
+```bash filename="Terminal"
+pnpm create zephyr-apps
+```
+</Tab>
+</Tabs>
+
+Follow the CLI prompts and select Parcel as your bundler. This will create a new project with all the necessary dependencies and configuration.
+
+## Manually
+
+<Tabs values={[{label:"npm"}, {label: 'yarn'}, {label: 'pnpm'}]}>
+<Tab>
+```bash filename="Terminal"
+mkdir my-parcel-app
+cd my-parcel-app
+npm init -y
+npm install parcel parcel-reporter-zephyr@latest react react-dom @types/react @types/react-dom typescript
+```
+</Tab>
+<Tab>
+```bash filename="Terminal"
+mkdir my-parcel-app
+cd my-parcel-app
+yarn init -y
+yarn add parcel parcel-reporter-zephyr@latest react react-dom @types/react @types/react-dom typescript
+```
+</Tab>
+<Tab>
+```bash filename="Terminal"
+mkdir my-parcel-app
+cd my-parcel-app
+pnpm init
+pnpm add parcel parcel-reporter-zephyr@latest react react-dom @types/react @types/react-dom typescript
+```
+</Tab>
+</Tabs>
+
+## Configure Zephyr with Parcel
+
+Create a `.parcelrc` file in your project root with the following configuration:
+
+```json filename=".parcelrc"
+{
+  "extends": "@parcel/config-default",
+  "reporters": [
+    "...",
```

Comment:
Idk if this is the standard or not but putting as a comment here is more clear imho, like:

```suggestion
    // ... other reporters,
```

Reply:
No comments allowed on json 🫠

Reply:
I know!
That's precisely the point hahahha
When code gets pasted, it will indicate an error. Ellipsis string will not.